ApeWorX/ape
# test: lessen amount of logs in ape's tests

### What I did

was getting too many unhelpful INFO logs in the test output, made it hard to see the reason for failures

### How I did it

remove a couple set level INFOs to set level ERRORs seems to do the trick in a conftest construct

### How to verify it

make a test fail on purpose, run tests, see the infos are no longer captured and theres way less output

### Checklist
